qatrfm/environment.py

```diff
@@ -32,23 +32,18 @@
 from qatrfm.utils import libutils
 
 
-class TerraformEnv(object):
+class TerraformCmd:
 
     logger = QaTrfmLogger.getQatrfmLogger(__name__)
 
-    def __init__(self, net_octet, tf_vars, tf_file=None, snapshots=False):
-        """Initialize Terraform Environment object."""
```

Comment:
I would like to keep description of the class, maybe instead of __init__ you can move it above, right after class statement.